pkg/ebpf/tracer_windows.go

```diff
@@ -154,10 +158,15 @@ func (t *Tracer) resizeBuffer(compareSize int, buffer []network.ConnectionStats)
 	return buffer
 }
 
-// getConnections returns all of the active connections in the ebpf maps along with the latest timestamp.  It takes
-// a reusable buffer for appending the active connections so that this doesn't continuously allocate
-func (t *Tracer) getConnections(active []network.ConnectionStats) ([]network.ConnectionStats, uint64, error) {
-	return nil, 0, ErrNotImplemented
+func (t *Tracer) resizeDriverBuffer(compareSize int, buffer []uint8) []uint8 {
```

Comment:
This and `resizeConnectionStatsBuffer` have basically the same code. Make them call one function?

Reply:
One important distinction now, is that the driver buffer has a preset length, but the stats buffers only a preset capacity.

Reply:
Unfortunately I can't because go doesn't have generics, unless you think I should try to do something funky with reflect/interfaces.

Reply:
If it was not for how `len` is handled, we could probably write something at least to calculate the new size of the buffer, but as @brycekahle mentions that is an important difference here. So its good as it is.